### PR TITLE
Some typos that flew under the radar (#527 fixes)

### PR DESCRIPTION
**Pull request by democat3457**

### CODE REVIEW

File: docs/1.12/content/Vanilla/Items/Item_Conditions.md [hunk 1]
```diff
@@ -94,7 +93,11 @@ item.withTag(NBTTag);
 
 You can also add your own itemConditions. These are special functions that accept the [item](/Vanilla/Items/IItemStack/) itself as single parameter.
 ```zenscript
-conditionedItem = item.only(function(item) {return true;});
+conditionedItem = item.only(function(itm as IItemStack) as bool {
+  var b as bool = true;
+  // do something with itm
```
Comment (Witixin1512) [inline docs/1.12/content/Vanilla/Items/Item_Conditions.md:98]: item*

Comment (democat3457) [reply docs/1.12/content/Vanilla/Items/Item_Conditions.md:98]: I used `itm` so as to not conflict with the `item` outside of the function (does this matter?)

Comment (Witixin1512) [reply docs/1.12/content/Vanilla/Items/Item_Conditions.md:98]: Oh true, apologies
I wonder what would be the best way to express that the one inside of the function is the predicate.

Comment (Witixin1512) [reply docs/1.12/content/Vanilla/Items/Item_Conditions.md:98]: Maybe use itemStack?

Comment (democat3457) [reply docs/1.12/content/Vanilla/Items/Item_Conditions.md:98]: that would work

**Pull request merged**